src/python/CRABInterface/Regexps.py

```diff
@@ -141,3 +141,7 @@
 
 # CUDAVersion style,  i.e. 11.4, 515.43.04
 RX_CUDA_VERSION = re.compile(r"^\d+\.\d+(\.\d+)?$")
+
+# UUID format of block name
+# 3517e1b6-76e3-11e7-a0c8-02163e00d7b3
+RX_BLOCK_UUID = re.compile(r"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$")
```

Comment:
is RX_BLOCK_UUID really used/needed ? Our "authority" on naming is WMCore/Lexicon.py which is happy with `BLOCK_STR = {'re': '#[a-zA-Z0-9\.\-_]+', 'maxLength': 100}`

Reply:
Sorry, I forget to remove. It is not use anymore because this PR only accept full block name.